taskcluster/ci/build/kind.yml

```diff
@@ -20,3 +20,4 @@ jobs-from:
     - android.yml
     - macos.yml
     - wasm.yml
+    - addons.yml
```

Comment:
I might reconsider rolling this into the build kind. My gut feeling is down the road keeping it as a separate kind will end up being easier to maintain. But up to you!